db/migrate/20210107092746_create_materialized_patient_summaries.rb

```diff
@@ -0,0 +1,5 @@
+class CreateMaterializedPatientSummaries < ActiveRecord::Migration[5.2]
+  def change
+    create_view :materialized_patient_summaries, materialized: true
```

Comment:
Building this for the first time takes ~30-40s on the perf box. The migration will refresh it once synchronously.